lib/dk_buttons_and_leds/Kconfig

```diff
@@ -21,6 +21,8 @@ config DK_LIBRARY_DYNAMIC_BUTTON_HANDLERS
 
 config DK_LIBRARY_BUTTON_NO_ISR
 	bool "Poll buttons unconditionally (no interrupts) [EXPERIMENTAL]"
+	# Workaround for buttons on nRF54L15 PDK in revision 0.2.x.
+	default y if BOARD_NRF54L15PDK_NRF54L15_CPUAPP && (BOARD_REVISION = "0.2.0" || BOARD_REVISION = "0.2.1")
```

Comment:
`default y if BOARD_NRF54L15PDK_NRF54L15_CPUAPP && (BOARD_REVISION = "0.2.0")`?
Seems like Zephyr generalizes to 2 revisions: https://github.com/nrfconnect/sdk-zephyr/blob/main/boards/arm/nrf54l15pdk_nrf54l15/revision.cmake

Reply:
Yes, but there is also physically revision 2.0.1 so added it to be safe if someone change it in upstream